### DISCUSSION TITLE
Should we add a "VERSION" constant?

### DISCUSSION THREAD

**Discussion by PaulRBerg** [sablier-labs/evm-monorepo#762] [Ideas]
Similar to the [version constant](https://github.com/PaulRBerg/prb-proxy/blob/28fbc73c6cf4180305bca5ff673502e531d99ea0/src/PRBProxyRegistry.sol#L35) in PRBProxy.

Related: https://github.com/sablier-labs/v2-core/issues/761

**Comment by PaulRBerg** [comment]
Just realized that this is potentially confusing because of the [npm package version](https://github.com/sablier-labs/v2-core/discussions/476).

**Reply by andreivladbrg** [reply]
> Just realized that this is potentially confusing

got to admit for an external user the package version(1.1.2) vs V2.1 is confusing

**Reply by PaulRBerg** [reply]
It might be, but there's no better solution I can think of.

@razgraf and I held a fruitful debate (i.e., a debate with lots of disagreements) about this topic in https://github.com/sablier-labs/v2-core/discussions/476  - I welcome any further feedback or criticism from you.

**Comment by smol-ninja** [comment]
This will be fixed after [Package tethering](https://github.com/sablier-labs/v2-core/issues/808) is merged.
